app/acme_directory_configuration.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+from urllib.parse import ParseResult as URLParseResult
+
+
+@dataclass
+class ACMEDirectoryConfiguration:
+    """
+    This data is generated from the /acme/directory endpoint. These endpoints can be different per server
+    """
+
+    new_order_url: URLParseResult
+    new_account_url: URLParseResult
+    new_nonce_url: URLParseResult
+    revoke_cert_url: URLParseResult
```

Comment:
I think these do not have to be parsed and can just be strings. With that we can also get rid of the `.geturl()` when using the endpoints.